bin/run_tests.py

```diff
@@ -28,7 +28,7 @@
     # unit tests
     unit_test_args = [sys.executable, "-m", "pytest", "unit_test"]
 
-    if sys.platform.startswith("linux"):
+    if sys.platform.startswith("linux") and os.environ.get("CIBW_PLATFORM", "linux") == "linux":
```

Comment:
This is to prevent running docker/podman tests in the pyodide test job which runs on Linux.
The alternative would be to setup Qemu in the pyodide test job.

Reply:
Oh I see, I had missed that the way to run pyodide tests is to set `CIBW_PLATFORM=pyodide`. That's maybe a little 'hidden' but that's not an issue for this PR - we defo don't need to run the qemu tests in that mode.